elements/rh-band/src/rh-band.scss

```diff
@@ -0,0 +1,208 @@
+@import "../../rh-sass/rh-sass";
+
+$LOCAL: band;
+$SLOT: "";
+$aside-md: 250px;
+$aside-lg: 300px;
+
+:host {
+  // Spacing values
+  --rh-band--paddingTop:                      calc(#{rh-var(container-spacer)} * 4);
+  --rh-band--paddingRight:                    #{rh-var(container-spacer)};
+  --rh-band--paddingBottom:                   calc(#{rh-var(container-spacer)} * 4);
+  --rh-band--paddingLeft:                     #{rh-var(container-spacer)};
```

Comment:
could we not combine these into one band-padding var?

Reply:
+1